docs/api/renderer.lua

```diff
@@ -41,10 +43,22 @@ function renderer.font.load(path, size, options) end
 ---
 ---Clones a font object into a new one.
 ---
+---@param font renderer.font
 ---@param size? number Optional new size for cloned font.
 ---
 ---@return renderer.font
-function renderer.font:copy(size) end
+function renderer.font.copy(font, size) end
+
+---
+---Sets the metatable of the given fonts array into that of renderer.font to
```

Comment:
Do we need the first part of the phrase? I think that it could confuse users a bit.

Reply:
Yes, when I was writing it, that was the last part I added thinking that I should be more explicit of what really happens so the user has a hint that the returned value is just an array of fonts with the font metatable. My reasoning for adding that comment came from the experience with the draw styled text function of widgets which didn't worked because the font group wasn't a userdata but an array (table) of fonts.

Reply:
The thing is that this is just an implementation detail and could change in the future.
I don't think we rely on the `fontgroup` being a table on the Lua side, and we also don't want users to touch its content.

Reply:
removed the line and force pushed the change